### DISCUSSION TITLE
Is this working on new Apple Silicon?

### DISCUSSION THREAD

**Discussion by niksmac** [elrumo/macOS_Big_Sur_icons_replacements#1138] [General]
Hey,
I am not able to replace icons in new [Apple Silicon](https://appleinsider.com/inside/apple-silicon) Mac. Is there any workaround?

**Comment by Scr1pting** [comment]
Replacing the icons should work on [Apple Silicon](https://appleinsider.com/inside/apple-silicon). Are you trying to replace the icons of system applications?

**Reply by niksmac** [reply]
No, I have tried this for atom.

**Reply by rodgerwerner** [reply]
Is this 'Atom' the open source txt editor? I just downloaded it onto my Mac mini M1, and changed the icon by dragging a new one on top of the icon in the 'info' window when I right click on the app.

**Reply by niksmac** [reply]
Yes, I got it working. May be I have tried something else.

**Reply by cake03** [reply]
Changing icons is the same irrespective of any version of Mac OS or the processor, provided the computer's software is too old that Finder doesn't have that feature
